service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java

```diff
@@ -1,20 +1,69 @@
 package bio.terra.cbas.controllers;
 
 import bio.terra.cbas.api.RunsApi;
+import bio.terra.cbas.config.CromwellServerConfiguration;
 import bio.terra.cbas.model.RunState;
 import bio.terra.cbas.model.RunStateResponse;
+import cromwell.client.ApiClient;
+import cromwell.client.api.WorkflowsApi;
 import java.util.UUID;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 
 @Controller
 public class RunsApiController implements RunsApi {
 
+  private final CromwellServerConfiguration cromwellConfig;
+
+  @Autowired
+  public RunsApiController(CromwellServerConfiguration cromwellConfig) {
+    this.cromwellConfig = cromwellConfig;
+  }
+
   @Override
   public ResponseEntity<RunStateResponse> postRun(String workflowUrl, Object workflowParams) {
+
+    ApiClient client = new ApiClient();
+    client.setBasePath(this.cromwellConfig.baseUri());
+    client.setDebugging(true);
+    client.setLenientOnJson(true);
+    WorkflowsApi workflowsApi = new WorkflowsApi(client);
     String runId = UUID.randomUUID().toString();
-    return new ResponseEntity<>(
-        new RunStateResponse().runId(runId).state(RunState.QUEUED), HttpStatus.CREATED);
+
+    ResponseEntity result;
+
+    try {
+      workflowsApi.submit(
+          "v1",
+          null,
+          workflowUrl,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null);
```

Comment:
Java doesn't have optional parameters. In Java the usual way to do this is either a builder pattern, for object construction, or a method override, which is what could work well here. If this was human written code (vs machine written code) then it could provide a nicer interface.

Looking at this API, if it instead accepted a single object that had optional fields (aka `WorkflowSubmission`) then this wouldn't be an issue.

Reply:
This is coming from a place of naivete (both in terms of context, and the fact that I don't know Java all that well), but: what about passing in an object as the sole argument? Then at least all these `null`s would get a name.

Reply:
> what about passing in an object as the sole argument?

Java doesn't work that way; you can't substitute an object for a parameter list (with the exception of an array argument and a varargs parameter list). You might be able to fix this by rewriting the swagger XML that generated this code, or by using a different client code generator.

Reply:
Yeah, the signature of `submit()` is auto-generated by the library that turns Cromwell Swagger into a Java library.

I think I should keep auto-generation changes out of the scope of this changeset.

Reply:
Would it make sense to have named parameters here?

Reply:
I don't think Java has named parameters either.

Reply:
> I don't think Java has named parameters either.

Yes, Java only has positional parameters. In Java, method overloading is used to implement optional or default parameters. https://docs.oracle.com/javase/tutorial/java/javaOO/methods.html